src/main/java/frc/robot/subsystems/Lights.java

```diff
@@ -0,0 +1,142 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class Lights extends SubsystemBase {
+  private LightCodes currentLightStatus = LightCodes.OFF;
+  private AddressableLED m_led = new AddressableLED(RobotMap.LED_PWM_PORT);
+  private AddressableLEDBuffer m_ledBuffer;
+  private int spacing = 1;
```

Comment:
Does `1` mean every `1` light is on, or `1` space between every on light? Feel free to clarify in a comment

Reply:
We added a comment as well, but a spacing value of 1 would mean every LED is on